controller/users.go

```diff
@@ -84,7 +84,7 @@ func (c *UsersController) Create(ctx *app.CreateUsersContext) error {
 	isSvcAccount := token.IsSpecificServiceAccount(ctx, "online-registration")
 	if !isSvcAccount {
 		log.Error(ctx, nil, "account used to call create api is not a service account")
-		return jsonapi.JSONErrorResponse(ctx, errors.NewForbiddenError("account not authorized to create users."))
+		return jsonapi.JSONErrorResponse(ctx, errors.NewUnauthorizedError("account not authorized to create users."))
```

Comment:
Any specific reason to use 401 vs 403 here?

Reply:
```
func IsSpecificServiceAccount(ctx context.Context, name string) bool {
	accountName, ok := extractServiceAccountName(ctx)
	if !ok {
		return false
	}
	return accountName == name
}
```
There can be two conditions for the ` IsSpecificServiceAccount` to be false:
1. because it's a user token ( which is ~ 401 )
2. because it's a service account token but doesn't belong to `online-registration`  ( which is definitely 403 )

I wanted to hide out this detail externally - and do a blanket 401.